src/test/kotlin/apply/application/ApplicationFormServiceTest.kt

```diff
@@ -233,4 +260,41 @@ class ApplicationFormServiceTest {
             )
         }
     }
+
+    @Test
+    fun `작성하지 않은 문항이 존재하는 경우 제출할 수 없다`() {
+        every { recruitmentRepository.findByIdOrNull(any()) } returns recruitment
+        every { applicationFormRepository.existsByApplicantIdAndSubmittedTrue(any()) } returns true
+        every { applicationFormRepository.findByRecruitmentIdAndApplicantId(any(), any()) } returns applicationForm1
+        every { recruitmentItemRepository.findByRecruitmentIdOrderByPosition(any()) } returns recruitmentItems
```

Comment:
`Mockk`의 최대 맹점이라고 생각되는 부분이네요.

`Mockito`의 경우에는 `when` ~ `thenReturn`으로 이어지는 부분에서 Stubbing + Verifying이 되는데
`Mockk`는 `every` ~ `returns`에서 Stubbing만 돼서 **실행되지 않는 로직이 있는 경우**에도 테스트가 통과해 버립니다.
실제로 지금은 `every { applicationFormRepository.existsByApplicantIdAndSubmittedTrue(any()) } returns true`에서 테스트가 끝나버려요. 
```suggestion
        every { applicationFormRepository.existsByApplicantIdAndSubmittedTrue(any()) } returns false
        every { applicationFormRepository.findByRecruitmentIdAndApplicantId(any(), any()) } returns applicationForm1
        every { recruitmentItemRepository.findByRecruitmentIdOrderByPosition(any()) } returns recruitmentItems
```

Reply:
신기하네요 👍 
해당하는 부분을 제대로 동작하도록 수정하고 추가로 message까지 확인하도록 수정했습니다 :)